### PR TITLE
rename folder from example to examples

### PR DESCRIPTION
**Pull request by supastore**
## What kind of change does this PR introduce?

Renamed example to examples
Updated .npmignore

Some formatting changes happened just by renaming the folder

### CODE REVIEW

**Comment by supastore** [comment]
closes #620

File: .npmignore [hunk 1]
```diff
@@ -16,7 +16,6 @@ yarn.lock
 src
 test
 examples
-example
 example-next-js
```
Comment (ibrahim-ahmed-dev) [inline .npmignore:19]: I dont think these are used too

Comment (ibrahim-ahmed-dev) [reply .npmignore:19]: going to resolve this comment because it may be out of scope of this task 👍

**Comment by soedirgo** [comment]
Thanks!

**Review by soedirgo — APPROVED** [review]

**Pull request merged**